src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const accessibilityQuiz = [
+  {
+    Question:
+      "The correct location to identify the main language of an entire page is:",
```

Comment:
I think this would work better as a question

```suggestion
      "Which one is the correct HTML element used to identify the main language of an entire page?",
```

Reply:
Thanks for the suggestion. I will implement it immediately.